helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -645,7 +645,7 @@ public boolean isLeader() {
 
     if (_helixPropertyStore == null) {
       String path = PropertyPathConfig.getPath(PropertyType.PROPERTYSTORE, _clusterName);
-      String fallbackPath = String.format("/%s/%s", _clusterName, "HELIX_PROPERTYSTORE");
```

Comment:
This is kept for backward compatibility, we need this when users upgrade from 0.6.2 to newer versions